src/apps/user/forms.py

```diff
@@ -1,6 +1,9 @@
 from allauth.account.forms import SignupForm
```

Comment:
Reorder imports:
0, default packages
1. django
2. frameworks
3. project files

Reply:
no changes?

Reply:
I also just read the PEP 8 and cant find the reference that django should go first before other libraries
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.

Reply:
because other packages/frameworks can relate to django, meaning it is better to import django first
it is not pep, it is Django best practice)

Reply:
Fair, now I understand, changes done))

Reply:
but if u can provide a doc or smth to read about it that would be great. I always relied on isort, so this is new to me, and i want to now how/where to google/search for such stuff

Reply:
cos now i am curios about FastAPI for example)

Reply:
Example from life

we had a package X and django

X internally imported django and patched some classes (internally!)

as a result - strange behaviour